samples/DotNet/Azure.Messaging.EventHubs/AnomalyDetector/Producer/Program.cs

```diff
@@ -0,0 +1,106 @@
+
+            // send messages to the event hub
+            await SendMessagesToEventHubAsync(1000);
+
+            await producerClient.CloseAsync();
+
+            Console.WriteLine("Press [enter] to exit.");
+            Console.ReadLine();
+
+            return 0;
+        }
+
+        // Creates an Event Hub client and sends messages to the event hub.
+        private static async Task SendMessagesToEventHubAsync(int numMessagesToSend)
+        {
+            var eg = new EventGenerator();
+
+            IEnumerable<Transaction> transactions = eg.GenerateEvents(numMessagesToSend);
+            
+            if (File.Exists(TransactionsDumpFile))
+            {
+                // exceptions not handled for brevity
+                File.Delete(TransactionsDumpFile);
+            }
+
+            File.AppendAllText(
+                TransactionsDumpFile, 
+                $"CreditCardId,Timestamp,Location,Amount,Type{Environment.NewLine}");
+
+            foreach (var t in transactions)
+            {
+                try
+                {
+                    // we don't send the transaction type as part of the message.
+                    // that is up to the downstream analytics to figure out!
+                    // we just pretty print them here so they can easily be compared with the downstream
+                    // analytics results.
+                    var message = t.Data.ToJson(); 
+
+                    if (t.Type == TransactionType.Suspect)
+                    {
+                        var fc = Console.ForegroundColor;
+                        Console.ForegroundColor = ConsoleColor.Yellow;
+
+                        Console.WriteLine($"Suspect transaction: {message}");
+
+                        Console.ForegroundColor = fc; // reset to original
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Regular transaction: {message}");
+                    }
+
+                    var line = $"{t.Data.CreditCardId},{t.Data.Timestamp.ToString("o")},{t.Data.Location},{t.Data.Amount},{t.Type}{Environment.NewLine}";
+
+                    File.AppendAllText(TransactionsDumpFile, line);
+
+                    // prepare a batch of events to send to the event hub. only one event in this case. 
+                    EventDataBatch eventBatch = await producerClient.CreateBatchAsync();
+                    eventBatch.TryAdd(new EventData(Encoding.UTF8.GetBytes(message)));
+
+                    // send the message to the event hub using the producer object
+                    await producerClient.SendAsync(eventBatch);
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine($"{t.ToJson()}{Environment.NewLine}Exception: {ex.Message}");
+                }
+
+                await Task.Delay(10);
+            }
+
+            Console.WriteLine($"{numMessagesToSend} messages sent.");
```

Comment:
>numMessagesToSend [](start = 33, length = 17)

This is intended count of messages. Can we count messages those were sent successfully and print it here?